app/views/candidate_mailer/application_rejected_all_rejected.erb

```diff
@@ -1,3 +1,37 @@
+<% if FeatureFlag.active?('apply_again') %>
```

Comment:
We should use separate templates for these too, it's quite tricky to see where the email starts/stops.